Server/SQL/procedures.sql

```diff
@@ -272,7 +272,8 @@ BEGIN
         L.last_update_device,
         Deck.share_code,
         User.name AS author,
-        L.accession
+        L.accession,
```

Comment:
Yea I was planning on having #156 merge first, and just rebase and fix the conflict here after